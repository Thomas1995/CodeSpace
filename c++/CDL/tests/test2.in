Testam cazuri particulare de intersectie

Vreau un dreptunghi in punctul (0,0) de laturi (3,2)
Vreau un dreptunghi in punctul (0,0) de laturi (3,2)

Muta in stanga figura 0 cu 2
Daca figura 1 intersecteaza figura 0
Vreau un poligon cu punctele (0,0), (3,3), (4,4)
Gata boss

Muta in stanga figura 0 cu 1
Daca figura 1 intersecteaza figura 0
Vreau un poligon cu punctele (1,1), (3,3), (4,4)
Gata boss

Muta in stanga figura 0 cu 1
Daca figura 1 intersecteaza figura 0
Vreau un poligon cu punctele (2,2), (3,3), (4,4)
Gata boss

Muta in dreapta figura 0 cu 4
Muta in sus figura 0 cu 2
Daca figura 1 intersecteaza figura 0
Vreau un poligon cu punctele (5,5), (3,3), (4,4)
Gata boss

Muta in sus figura 0 cu 1
Daca figura 1 intersecteaza figura 0
Vreau un poligon cu punctele (6,6), (3,3), (4,4)
Gata boss

Muta in jos figura 0 cu 1
Muta in stanga figura 0 cu 3
Daca figura 1 intersecteaza figura 0
Vreau un poligon cu punctele (7,7), (3,3), (4,4)
Gata boss

Muta in jos figura 0 cu 4
Daca figura 1 intersecteaza figura 0
Vreau un poligon cu punctele (8,8), (3,3), (4,4)
Gata boss

Muta in dreapta figura 0 cu 6
Daca figura 1 intersecteaza figura 0
Vreau un poligon cu punctele (9,9), (3,3), (4,4)
Gata boss

Muta in sus figura 0 cu 4
Daca figura 1 intersecteaza figura 0
Vreau un poligon cu punctele (-1,-1), (3,3), (4,4)
Gata boss
