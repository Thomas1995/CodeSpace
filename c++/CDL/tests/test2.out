0 Dreptunghi:
3 2
3 4
6 2
6 4
1 Dreptunghi:
0 0
0 2
3 0
3 2
2 Poligon:
0 0
3 3
4 4
3 Poligon:
1 1
3 3
4 4
4 Poligon:
3 3
4 4
5 5
5 Poligon:
3 3
4 4
7 7
6 Poligon:
3 3
4 4
8 8
7 Poligon:
3 3
4 4
9 9
8 Poligon:
-1 -1
3 3
4 4
